plexpy/config.py

```diff
@@ -410,4 +413,14 @@ def _upgrade(self):
             if self.VIDEO_LOGGING_ENABLE == 0:
                 self.MOVIE_LOGGING_ENABLE = 0
                 self.TV_LOGGING_ENABLE = 0
-            self.CONFIG_VERSION = '1'
\ No newline at end of file
+            self.CONFIG_VERSION = '1'
+        if self.CONFIG_VERSION == '1':
```

Comment:
This tries to run even if there is no config.ini file causing the program to terminate on first run.